styles.css

```diff
@@ -111,8 +111,694 @@ nav i {
   margin-left: 15px;
 }
 
-figure {
+.fig1 {
   position: absolute;
   top: 470px;
   right: -40px;
 }
+
+main {
+  display: grid;
+}
+
+.my-recent-works {
+  background: #f5f5f5;
+  display: flex;
+  flex-direction: column;
+  left: 500px;
+  top: 700px;
```

Comment:
Kindly fix the display of the page and you'll be good to go. The website seems to have some positioning issues as I can see that you add height, top and left positions to most of the sections in order to position them. You need to allow the elements to flow from top to bottom rather than manually assigning top and left positions.
  Here's how it looks currently:
![image](https://user-images.githubusercontent.com/20554583/192127056-64d06b63-9929-4769-bfa2-7a3a50333982.png)